docs/content/tutorial/custom-steps.md

```diff
@@ -0,0 +1,154 @@
+# Custom steps for JIRA
+
+In this tutorial, you'll learn how to configure custom steps for use with JIRA. Here's what we'll do with this sample app:
+
+1. Create your app from an app manifest and clone a starter template
+2. Set up and run your local project
+3. Create a workflow using Workflow Builder with custom steps
+4. Create an issue in JIRA using your custom step
+
+## Prerequisites {#prereqs}
+
+Before getting started, you will need the following:
+
+* a development workspace where you have permissions to install apps. If you don’t have a workspace, go ahead and set that up now&mdash;you can [go here](https://slack.com/get-started#create) to create one, or you can join the [Developer Program](https://api.slack.com/developer-program) and provision a sandbox with access to all Slack features for free.
+* a development environment with [Python 3.6](https://www.python.org/downloads/) or later.
+
+**Skip to the code**
+If you'd rather skip the tutorial and just head straight to the code, you can use our [Bolt for Python JIRA functions sample](https://github.com/slack-samples/bolt-python-jira-functions) as a template.
+
+## Creating your app {#create-app}
+
+1. Navigate to the [app creation page](https://api.slack.com/apps/new) and select **From a manifest**.
+2. Select the workspace you want to install the application in, then click **Next**.
+3. Copy the contents of the [`manifest.json`](https://github.com/slack-samples/bolt-python-ai-chatbot/blob/main/manifest.json) file into the text box that says **Paste your manifest code here** (within the **JSON** tab) and click **Next**.
+4. Review the configuration and click **Create**.
+5. You're now in your app configuration's **Basic Information** page. Click **Install App**, then **Install to _your-workspace-name_**, then **Allow** on the screen that follows.
+
+### Obtaining and storing your environment variables {#environment-variables}
+
+Before you'll be able to successfully run the app, you'll need to obtain and set some environment variables.
+
+1. On the **Install App** page, copy your **Bot User OAuth Token**. You will store this in your environment as `SLACK_BOT_TOKEN` (we'll get to that next).
```

Comment:
May want to mention here that you have to install the app to the workspace for the token to show up